### PR TITLE
doc: improve windows build instructions

### PR DESCRIPTION
**Pull request by alonbl**

### CODE REVIEW

**Comment by alonbl** [comment]
@gilweis

**Comment by gilweis** [comment]
***@***.***:~/tmp/tmp1/mycms$ ./Configure --prefix="${MYCMS_PREFIX}"
--cross-compile-prefix=x86_64-w64-mingw32- mingw64
bash: ./Configure: No such file or directory

***@***.***:~/tmp/tmp1/mycms$ ./configure --prefix="${MYCMS_PREFIX}"
--cross-compile-prefix=x86_64-w64-mingw32- mingw64
configure: error: unrecognized option:
`--cross-compile-prefix=x86_64-w64-mingw32-'
Try `./configure --help' for more information

File: INSTALL.md [hunk 1]
```diff
@@ -81,13 +81,31 @@ $ dpkg -i mycms*.deb
...
-$ ./configure --host=x86-w64-mingw32 --prefix=/ ...
-$ make install DESTDIR="$(pwd)/tmp"
-$ DESTDIR=tmp ./packaging/windows-nsis/build
+$ set MYCMS_PREFIX=/tmp/mycms-prefix
+```
+
+#### Openssh
+
+```
+$ ./Configure --prefix="${MYCMS_PREFIX}" --cross-compile-prefix=x86_64-w64-mingw32- mingw64
```
Comment (alonbl) [inline INSTALL.md:100]: @gilweis : this should be run in openssl sources :)

File: INSTALL.md [hunk 2]
```diff
@@ -81,13 +81,35 @@ $ dpkg -i mycms*.deb
...
+```
+
+#### OpenSSL Build
+
+Download and extract OpenSSL tarball.
+
+Run the following in OpenSSL source directory:
+
+```
+$ ./Configure --prefix="${MYCMS_PREFIX}" --cross-compile-prefix=x86_64-w64-mingw32- mingw64
```
Comment (gilweis) [inline INSTALL.md:100]: /bin/bash ../../libtool --mode=compile --tag=RC  -DHAVE_CONFIG_H -I. -I../..  -I../../include  -DWIN32_LEAN_AND_MEAN -i "versioninfo.rc" -o "versioninfo.lo"
Usage: /home/g/tmp/tmp1/mycms/libtool [OPTION]... [MODE-ARG]...
Try 'libtool --help' for more information.
libtool:   error: unrecognised option: '-DHAVE_CONFIG_H'

Comment (alonbl) [reply INSTALL.md:100]: Have you installed `mingw-w64`? can you please send me config.log?

File: INSTALL.md (hunk 1 above)
Comment (gilweis) [reply INSTALL.md:100]: please change from:
$ set MYCMS_PREFIX=/tmp/mycms-prefix
to
$ MYCMS_PREFIX=/tmp/mycms-prefix

Comment (alonbl) [reply INSTALL.md:100]: Done

**Pull request closed without merging**